contract_show_invoice/views/contract_view.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<openerp>
+<odoo>
     <data>
```

Comment:
No need for the data tag anymore, can remove and decrease indentation level four spaces